pkg/bindings/test/common_test.go

```diff
@@ -113,7 +114,38 @@ func (b *bindingTest) startAPIService() *gexec.Session {
 }
 
 func (b *bindingTest) cleanup() {
+	s := b.runPodman([]string{"stop", "-a", "-t", "0"})
+	s.Wait(45)
```

Comment:
Seems like a long time to wait?

Reply:
It wont wait that long if it completes.  It will wait at least that long if it does not.